src/command_extract.cpp

```diff
@@ -165,14 +165,33 @@ static osmium::Box parse_bbox(const rapidjson::Value& value) {
     throw config_error{"'bbox' member is not an array or object."};
 }
 
+#ifdef _WIN32
+static bool isValidDriverChar(char value) {
```

Comment:
Can you please use names consistent with the naming everywhere else (`is_valid_driver_char` in this case. More cases like that below.

Reply:
Done